xDnsServer.psd1

```diff
@@ -48,9 +48,7 @@ PrivateData = @{
 
         # ReleaseNotes of this module
         ReleaseNotes = '* Unit tests updated to use standard unit test templates.
-* MSFT_xDnsServerZoneTransfer: Added unit tests.
-                               Updated to meet Style Guidelines.
-* MSFT_xDnsARecord: Removed hard coding of Localhost computer name to eliminate PSSA rule violation.
+* MSFT_xDnsRecord: Added property DnsServer to specify DnsServer instead of just running on localhost
```

Comment:
Please remove the update to the module manifest (.psd1 file).
The module manifest is updated from the release notes in the README.md or the CHANGELOG.md when the module is released.